WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSEligibilityChecker.swift

```diff
@@ -60,18 +55,6 @@ final class POSEligibilityChecker: POSEligibilityCheckerProtocol {
 }
 
 private extension POSEligibilityChecker {
-    var isOnboardingComplete: AnyPublisher<Bool, Never> {
-        return cardPresentPaymentsOnboarding.statePublisher
```

Comment:
`cardPresentPaymentsOnboarding` could be removed from the class.

Reply:
Great catch! Removed all related onboarding logic in POS eligibility check in https://github.com/woocommerce/woocommerce-ios/pull/14787/commits/a7abd66e222260c21de8d16aa7dc889bd2e9b8fd.